### PR TITLE
2015.10 stable

### PR DESCRIPTION
**Pull request by Ham22**
New(ish) uboot update which includes the 2015.10 release and our fitimage and tpm additions.

### CODE REVIEW

File: .gitreview [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[gerrit]
```
Comment (abhimanyuv1) [inline .gitreview]: I think these changes doesnt make sense here?

Comment (Ham22) [reply .gitreview]: Have removed the config by backing it out as didn't want to change the history of the base branch. Thoughts?

Comment (abhimanyuv1) [reply .gitreview]: Yup i  am fine with this.

**Comment by abhijit-mahajani-imgtec** [comment]
Approved. just re-create the pull request for a new branch 2015.10?

**Comment by Ham22** [comment]
Am closing the PR and shall just push the new branch directly

**Pull request closed without merging**